.github/workflows/unfolding.yml

```diff
@@ -0,0 +1,457 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Unfolding
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  pull_request:
+    branches: [ main ]
+  schedule:
+    - cron: '0 3 * * 1,3,5' # Run on Monday, Wednesday, and Friday morning at 3h00 UTC
+    - cron: '30 5 * * 1-5' # Run on weekdays
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+env: 
+  MAX_FILES: 10
+  LUMI_SCALE: 760
+  NTHREADS: 16
+  LOCAL_WEB_DIR: www/WMassAnalysis/PRValidation
```

Comment:
Is there any way to store common variables like this in a common file that both workflows read from?

Reply:
Yes there is the possibility to write custom "actions" and I wasted some hour with this already. I was not able to get the outputs from the custom actions so in the end I just copied the code. But my attempt is still there in an older commit, see:
https://github.com/davidwalter2/WRemnants/blob/2f1b6e5acc930f925824601d903bb737b5398c3c/.github/actions/setenv/action.yml

Reply:
Ok fair, can waste more time in the future when we are motivated